docs/react.md

```diff
@@ -18,15 +18,16 @@ docker-compose up -d --build
 ```
 
 1. Switch to the `react` directory.
-1. Create a `.env` with these parameters that you can change as needed.
+2. Create a `.env` with these parameters that you can change as needed.
 
    ```
    REACT_APP_NAME=Stager
    REACT_APP_MINIO_URL=http://localhost:9000/
    ```
 
-1. Install dependencies with `yarn`.
-1. Start a development server in watch mode with `yarn start`.
+3. Set ```ENABLE_OIDC=``` to blank (empty).
```

Comment:
This step applies to the steps in `docker.md` as it concerns the backend. You also do not need to actually number in order for Markdown ordered lists.

Reply:
I can add it there as well. But I think this should be here as well. The current doc says that if the user does all the given steps, react app should work which isn't true. What do think?

Reply:
The docs here are scoped specifically to the frontend. It links to `docker.md` before the steps for how to set that up. I think the problem is that as we discovered at the end of last week, the docs each only describe a component, and there's no "Getting started" section in the main README to elucidate how to read them.